src/components/BrowseArticles.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useState } from "react";
+import { getArticles, getArticlesTopics } from "./Axios";
+import { Link, useParams } from "react-router-dom";
+import Voter from "./Voter";
+import { dateChanger } from "../../utils/utils";
+
+export default function BrowseArticles({ sortBy, sortAsc }) {
+  const { topic } = useParams();
+  const [isLoading, setIsLoading] = useState(true);
+
+  const [articles, setArticlesInfo] = useState([]);
+
+  function filterSort(articles, sortBy, sortAsc) {
+    const sortedArticles = articles.slice().sort((a, b) => {
```

Comment:
You shouldn't need to sort manually here. Remember that your api should accept sort by queries which we can access when making the api call